.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": "eslint:recommended",
+  "overrides": [],
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "rules": {
+    "no-duplicate-imports": 1,
```

Comment:
Why these rules?

Reply:
I chose to use 'no-duplicate-imports' because I couldn't of a reason for duplicate imports to be necessary.

I chose to implement the for loop enforcer because a for loop function with one iteration could be written more efficiently.